gdk/commands/component/transformer/BuildRecipeTransformer.py

```diff
@@ -31,6 +33,15 @@ def transform(self, build_folders):
             raise Exception(RECIPE_SIZE_INVALID.format(self.project_config.recipe_file, input_recipe_file_size))
 
         component_recipe = CaseInsensitiveRecipeFile().read(self.project_config.recipe_file)
+
+        logging.info("Validating the recipe against the Greengrass recipe schema.")
+        try:
+            recipe_schema_path = utils.get_static_file_path(consts.recipe_schema_file)
+            validator = RecipeValidator(recipe_schema_path)
+            validator.validate_recipe(component_recipe.to_dict())
+        except jsonschema.exceptions.ValidationError as err:
```

Comment:
bypass-able?

Reply:
Not implemented to by bypassable, since the cloud behavior is going to be enforced and not opt-out